tianshou/policy/modelfree/a2c.py

```diff
@@ -105,11 +105,12 @@ def learn(self, batch: Batch, batch_size: int, repeat: int,
             for b in batch.split(batch_size):
                 self.optim.zero_grad()
                 dist = self(b).dist
-                v = self.critic(b.obs)
+                v = self.critic(b.obs).squeeze()
```

Comment:
You should specify the axis if possible.

Reply:
This is a tensor with shape `(bsz, 1)`. I'd like to squeeze all the shape.

Btw, another critical issue: Categorical distribution outputs [bsz] while Normal distribution outputs [bsz, 1], so at this time we cannot determine whether this log_prob tensor is squeezed or not. Thus I use squeeze() in all log_prob.

Reply:
There is no side effect to also squeeze `bsz` part ? because if any of the dim has size 1, it will be removed. Are you sure it cannot be an issue ?

Reply:
Okay.